<?xml version="1.0"?>

<launch>
    
    <param name="mode" value="connect_points" />
    <node name="robot_interface" pkg="njllrd_proj2" type="robot_interface.py"
        output="screen" />
  
    <node name="controller" pkg="njllrd_proj2" type="controller.py" 
        output="screen" />

  
</launch>

